src/languages/mathematica.js

```diff
@@ -76,29 +79,44 @@ export default function(hljs) {
     begin: /[+\-*/,;.:@~=><&|_`'^?!%]+/
   };
 
-  const PATTERNS_AND_SLOTS = {
-    className: 'pattern-match',
-    variants: [
-      {begin: /([a-zA-Z$][a-zA-Z0-9$]*)?_+([a-zA-Z$][a-zA-Z0-9$]*)?/},
-      {begin: /#[a-zA-Z$][a-zA-Z0-9$]*|#+[0-9]?/}
-    ]
+  const PATTERNS = {
+    className: 'pattern',
+    begin: /([a-zA-Z$][a-zA-Z0-9$]*)?_+([a-zA-Z$][a-zA-Z0-9$]*)?/,
+  };
+
+  const SLOTS = {
+    className: 'slot',
+    begin: /#[a-zA-Z$][a-zA-Z0-9$]*|#+[0-9]?/
   };
 
   const BRACES = {
+    className: 'brace',
     begin: /[[\](){}]/
   };
 
   const MESSAGES = {
-    className: 'string',
+    className: 'message-name',
     begin: regex.concat("::", SYMBOL_RE),
   };
 
   return {
     name: 'Mathematica',
     aliases: ['mma', 'wl'],
     disableAutodetect: true,
```

Comment:
We'll turn this back on when we're ready right, or did we discuss why this is a bad idea already?

Reply:
@joshgoebel It depends on if this screws up the unit-tests, right? If you say we should turn it on, then we turn it on.